dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityOptions.nlsprops

```diff
@@ -266,4 +266,50 @@ tlsProfiler.option-desc.file=\
 tlsProfiler.option-key.verbose=\
 \ \ \ \ --verbose
 tlsProfiler.option-desc.verbose=\
-\tInclude additional information about unsafe protocols.
\ No newline at end of file
+\tInclude additional information about unsafe protocols.
+
+# -------- importSigner OPTIONS ------------------------------------------------#
+# List of descriptive text for options. Use "\n\" for line continuations.
+# The code provides no whitespace: begin keys with "\ \ \ \ " for leading
+# whitespace, and begin descriptions with \t
+#------------------------------\n at 72 chars -------------------------\n\#
+#------------------------------\n at 72 chars Leading "\ \ \ \ "-------\n\#
+#------------------------------\n at 72 chars Leading "\t"-------------\n\#
+importSigner.desc=\
+\tImport the signer certificate of a target host and port into the provided keystore.
+importSigner.usage.options=\
+\t{0} importSigner [options]
+importSigner.option-key.password=\
+\ \ \ \ --password[=pwd]
+importSigner.option-desc.password=\
+\tKeystore password, minimum {1} characters.\n\
+\tIf no value is defined you will be prompted.\n\
+\tRequired if store type is PKCS12 or JKS.
+importSigner.required-key.keyStore=\
+\ \ \ \ --keyStore
+importSigner.required-desc.keyStore=\
+\tSpecify the KeyStore file.
+importSigner.option-key.port=\
+\ \ \ \ --port
+importSigner.option-desc.port=\
+\tSpecify the port. Defaults to 443
+importSigner.required-key.host=\
+\ \ \ \ --host
+importSigner.required-desc.host=\
+\tSpecify the host.
+importSigner.option-key.type=\
+\ \ \ \ --type
+importSigner.option-desc.type=\
+\tThe type of the keystore. If no value is provided, it is determined from the keystore file extension\n\
+\tAllowed values: PKCS, JKS, PEM.
+importSigner.option-key.alias=\
+\ \ \ \ --alias
+importSigner.option-desc.alias=\
+\tThe alias to store the certificate under. If no value is specified, it is generated.
+importSigner.option-key.accept=\
+\ \ \ \ --accept
+importSigner.option-desc.accept=\
+\tWhether to accept the cert or not. Allows non-interactive acceptance of the certificate into\n\
+\tthe keystore. If no value is specified the signer is imported with no check. If a value is\n\
```

Comment:
If no value is specified the signer is imported with no check
-->
If no value is specified, the signer is imported with no check